doc/Building_targets.md

```diff
@@ -0,0 +1,179 @@
+# Building targets
+
+KOReader is available for multiple platforms. Here are instructions to build installable packages for all these platforms.
+
+These instructions are intended for a Linux OS. MacOS and Windows users are suggested to develop in a Linux VM.
+
+
+## Prerequisites
+
+This instructions asume that you [have a development environment ready to run](Building.md) KOReader. If not then please install common prerequisites first.
+
+### A toolchain for your target.
+
+Each target has its own architecture and you'll need to setup a proper cross-compile toolchain. Your GCC should be at least version 4.8
```

Comment:
I just bumped that to 4.9 with https://github.com/koreader/koreader-base/pull/917
```suggestion
Each target has its own architecture and you'll need to setup a proper cross-compile toolchain. Your GCC should be at least version 4.9.
```